eng/scripts/Language-Settings.ps1

```diff
@@ -30,7 +30,7 @@ function Get-cpp-PackageInfoFromPackageFile($pkg, $workingDirectory)
   return New-Object PSObject -Property @{
     PackageId      = $pkgName
     PackageVersion = $pkgVersion
-    ReleaseTag     = "$($pkgName)_$($pkgVersion)"
+    ReleaseTag     = "${pkgName}_${pkgVersion}"
```

Comment:
What is this change doing exactly? Can you help me understand (I have little context and powershell expertise here).

Reply:
Honestly its not going to make a difference functionally. But I was not sure why the syntax was used in the original code. All am doing here is transferring the code as is from the original file where it was created.